crypto/key_manager.go

```diff
@@ -144,7 +144,7 @@ func parsePrivateKey(key []byte) (crypto.PrivateKey, trillian.SignatureAlgorithm
 	// Our two ways of reading keys are ParsePKCS1PrivateKey and ParsePKCS8PrivateKey.
 	// And ParseECPrivateKey. Our three ways of parsing keys are ... I'll come in again.
 	if key, err := x509.ParsePKCS1PrivateKey(key); err == nil {
-		return key, trillian.SignatureAlgorithm_ANONYMOUS, nil
+		return key, trillian.SignatureAlgorithm_RSA, nil
```

Comment:
I don't think it will be a problem -- Go only has parsing code for RSA/DSA/ECDSA private keys, and they all have known enum values for signatures.